src/alexa_go_vote/polling_place.cljs

```diff
@@ -6,71 +6,127 @@
             [cljs-lambda.context :as ctx])
   (:require-macros [cljs.core.async.macros :refer [go]]))
 
-(defn dialog-delegate
-  []
-  (.log js/console "Responding with Dialog.Delegate")
+(def dialog-delegate-response
   {:version 1.0
    :response {:directives [{:type "Dialog.Delegate"}]}})
 
-(defn server-error []
-  (.log js/console "Responding with server-error")
+(def no-info-response
   {:version 1.0
-   :response {:outputSpeech
-              {:type "PlainText"
-               :text "I'm sorry, we're experiencing a server problem. Goodbye."
-               :shouldEndSession true}}})
+   :response
+   {:outputSpeech
+    {:type "PlainText"
+     :text (str "I'm sorry, I couldn't find a result for that address. "
+                "Check with your local election official, or try again "
+                "closer to the election.")}
+    :card
+    {:type "Standard"
+     :title "No Results"
+     :text (str "You can try your search again at icantfindmypollingplace.com")}
+    :shouldEndSession true}})
 
-;; (query-polling-place "3632  INTERLAKE AVE N SEATTLE WA 98103")
-
-(defn process-response
+(defn response->clj
   [response]
-  (if (http/unexceptional-status? (:status response))
-    (let [body-json (->> response :body .toString (.parse js/JSON))
-          body (js->clj body-json :keywordize-keys true)
-          polling-locations (:pollingLocations body)
-          polling-location (first polling-locations)
-          address (get polling-location :address)
-          name (get address :locationName)
-          address (str/join ", " [(get address :line1) (get address :line2)
-                                  (get address :city) (get address :zip)])]
-      (.log js/console "Responding with polling location " name)
+  (as-> response $
+    (:body $)
+    (.toString $)
+    (.parse js/JSON $)
+    (js->clj $ :keywordize-keys true)))
+
+(defn body->polling-place-info
+  [body]
+  (.log js/console (str "Response body: " (pr-str body)))
+  (let [polling-locations (:pollingLocations body)
```

Comment:
Maybe a lot of this could get reduced to a single threaded expression + destructuring...?